tests/by-util/test_pidof.rs

```diff
@@ -33,3 +33,29 @@ fn test_no_program() {
 fn test_no_pid_found() {
     new_ucmd!().arg("NO_THIS_PROGRAM").fails().code_is(1);
 }
+
+#[test]
+#[cfg(target_os = "linux")]
+fn test_quiet() {
+    let binding = new_ucmd!().arg("kthreadd").arg("-q").succeeds();
+    let output = binding.stdout();
+
+    assert!(output.is_empty())
+}
+
+#[test]
+#[cfg(target_os = "linux")]
+fn test_s_flag() {
+    let binding = new_ucmd!()
+        .args(&["-s", "kthreadd", "kthreadd", "kthreadd"])
+        .succeeds();
+    let output = binding.stdout_str();
+
+    let binding = output.replace('\n', "");
+    let pids = binding.split(' ').collect::<Vec<_>>();
+    let first = pids[0];
+
+    let result = pids.iter().all(|it| *it == first);
+
+    assert!(result)
+}
```

Comment:
I think this test is a better fit for #124 and so I would remove it from this PR. I already left two comments in #124.